mewarpx/docs/development.rst

```diff
@@ -0,0 +1,65 @@
+.. _development-page:
+
+==============================
+Python package development use
+==============================
+
+This describes how to compile and install WarpX, pywarpx, and mewarpx in order
+to make development changes require minimal recompilation.
+
+Setup
+-----
+
+.. code-block:: bash
+
+    # Create a dedicated build directory, outside the source tree
+    mkdir -p ~/build/build_tree_name
+    cd  ~/build/build_tree_name
+
+    # Setting this environment variable indicates to pywarpx that it should refer
+    # to an already built WarpX library rather than compiling it for itself
+    export PYWARPX_LIB_DIR=~/build/build_tree_name/lib/
+
+    # Set WarpX_COMPUTE as desired
+    # Set WarpX_LIBS=ON to generate the shared library that pywarpx will reference
+    # Pass -G Ninja to use the Ninja build system since it's somewhat faster than GNU Make
+    cmake ~/repos/WarpX/ -DWarpX_DIMS=2 -DWarpX_EB=ON -DWarpX_OPENPMD=ON \
+        -DWarpX_COMPUTE=CUDA \
+        -DWarpX_LIB=ON \
+        -G Ninja
+
+    # Run ninja to build
+    ninja
+
+
+    # Create a dedicated Python virtual environment
+    mkdir -p ~/venvs/venv_tree_name
+    cd  ~/venvs/venv_tree_name
+
+    python3 -m venv .
+    source bin/activate
+
+    # Install all of the pre-requisite Python packages
+    bash ~/repos/WarpX/mewarpx/medocker/docker_shared/python_packages.sh
+
+    # Install pywarpx
+    pip install -e ~/repos/WarpX/Python/
+
+    # Install mewarpx and minerva with full set of support libraries
+    pip install -e ~/repos/WarpX/mewarpx/[complete]
+    pip install -e ~/repos/minerva/[complete]
```

Comment:
`minerva` is not a dependency for `mewarpx` and since this is all public, I'd say we should remove this line:
```suggestion
```

Reply:
Will push with the comment corrected, too.